### PR TITLE
feat: Single product page

### PR DESCRIPTION
**Pull request by nayyyhaa**
## In this PR :
-  I've just added single product page.

#### Component files

- [x] [components/Card.js](https://github.com/nayyyhaa/inNotion-care-react/compare/single-product-page?expand=1#diff-7916963985f6b54523951aa9bdcdb855db5905bed40fc7775a485986aec0fa5d)
- [x] [components/CartCard.js](https://github.com/nayyyhaa/inNotion-care-react/compare/single-product-page?expand=1#diff-052126a8921eb24c8d036b50b456f25f263db9fbc51b55153175215de5afc297)
- [x] [components/Navbar.js](https://github.com/nayyyhaa/inNotion-care-react/compare/single-product-page?expand=1#diff-c2bbe5525aa57a971df65b233f8bbe05d6861f64b49bb9d5e0475a3d6ec44838)

#### Page
- [x] [pages/SingleProductPage.js](https://github.com/nayyyhaa/inNotion-care-react/compare/single-product-page?expand=1#diff-cd0956fd3590b8e9329ced019e120adaa4d7ac6131355484f60d22b1b6f8c60e)

#### utils:
- [x] [custom-hooks/usePageViewTracker.js](https://github.com/nayyyhaa/inNotion-care-react/compare/single-product-page?expand=1#diff-50c269b376b3002a7b4cb1edd3ac18e25afac7c7d142c496220fa53713f10d9c)
 
#### files to *ignore*
- [ ] files with just few import changed
- [ ] CSS files

### Live preview : https://deploy-preview-7--innotion-carev2.netlify.app/

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
Amazing work as always . 
Your naming convention is so nice I really liked it a lot.
You can merge it. kudos to you

File: src/components/Navbar.js [hunk 1]
```diff
@@ -20,18 +22,28 @@ export const Navbar = () => {
...
+          <input
+            type="search"
+            className="input search-nav reset-ip p-05"
+            placeholder="Search here"
+            id="search-text"
+            value={searchIp}
+            onChange={(e) => setSearchIp(e.target.value)}
+          />
+          <Link to="/products">
+            <button className="btn secondary-outline-btn m-l-1">Search</button>
```
Comment (arjun-computer-geek) [inline src/components/Navbar.js:52]: is there need of button here? I think it is useless here right?

**Review by arjun-computer-geek — APPROVED** [review]
A small suggestion. Please consider if it is necessary. Else All codes are well structure.

**Review by VLeads — APPROVED** [review]
Nothing to suggest. Overall great job, Nice UI

File: src/components/Navbar.js (hunk 1 above)
Comment (nayyyhaa) [reply src/components/Navbar.js:52]: if a user is on a different page, the search button will redirect to the products page.

**Comment by nayyyhaa** [comment]
Thanks all for reviewing. Merging the changes.

**Pull request merged**